index.html

```diff
@@ -298,38 +298,38 @@ <h3 class="table-header">Saturday, October 30th</h3>
         <table class="table">
             <tr>
                 <td class="left">Midnight Snack</td>
-                <td class="middle">TBA</td>
+                <td class="middle">6th floor Commons</td>
                 <td class="right">12 AM</td>
```

Comment:
Convert all times to the same format for consistency, like 12:00AM if minutes are missing from any